lua/nvim-tree/renderer/init.lua

```diff
@@ -13,12 +13,13 @@ local M = {}
 local SIGN_GROUP = "NvimTreeRendererSigns"
 
 local namespace_id = vim.api.nvim_create_namespace "NvimTreeHighlights"
+local namespace_extmarks_id = vim.api.nvim_create_namespace "NvimTreeExtmarks"
```

Comment:
Nice find.

Could we play this a bit safer and have a separate namespace for [full-name](https://github.com/nvim-tree/nvim-tree.lua/blob/26632f496e7e3c0450d8ecff88f49068cecc8bda/lua/nvim-tree/renderer/components/full-name.lua#L69) as well?

It's using `"NvimTreeHighlights"` which is a little concerning now that we've learned of the namespace sharing difficulties.

Reply:
Would be nice to rename it too, indeed, doesn't have to happen here though.

Reply:
I looked at the the namespace usage in ``full_name.lua``, and it seems to rely on the ``NvimTreeHighlights`` namespace being set up beforehand. I thought this because of line 69:
```lua
local extmarks = vim.api.nvim_buf_get_extmarks(0, ns_id, { line_nr - 1, 0 }, { line_nr - 1, -1 }, { details = true })
```
Because of that I don't think I can introduce a separate namespace for full_name rn, also
I did not wrote any code in that file I'm not so sure what I could safely change.

Reply:
Fair enough; there may all sorts of assumptions there.